src/uu/nohup/src/nohup.rs

```diff
@@ -112,50 +110,39 @@ fn find_stdout() -> File {
         .write(true)
         .create(true)
         .append(true)
-        .open(Path::new("nohup.out"))
+        .open(Path::new(NOHUP_OUT))
     {
         Ok(t) => {
-            show_warning!("Output is redirected to: nohup.out");
+            show_info!("ignoring input and appending output to '{}'", NOHUP_OUT);
             t
         }
-        Err(e) => {
+        Err(e1) => {
             let home = match env::var("HOME") {
-                Err(_) => crash!(2, "Cannot replace STDOUT: {}", e),
+                Err(_) => {
+                    show_info!("failed to open '{}': {}", NOHUP_OUT, e1);
+                    exit!(125)
```

Comment:
Could you please document why 125?
Feels like a magic number

Reply:
From `info coreutils 'nohup'` :

```
Exit status:

     125 if ‘nohup’ itself fails, and ‘POSIXLY_CORRECT’ is not set
     126 if COMMAND is found but cannot be invoked
     127 if COMMAND cannot be found
```

Reply:
Yeah, I figured. I meant that this should be explicit in the code :)
```suggestion
                    //  To match GNU's exit code
                    exit!(125)
```

Reply:
I've added some error code defines from the C implementation to increase readability, along with treating that `POSIXLY_CORRECT` case, however It's still not 100% equivalent to the C implementation.

These defines should probably be moved into the utils module, maybe even placed behind some macros/functions.